### ISSUE TITLE
Prefix incorrect before first value change

### ISSUE THREAD

**Issue by henriquecm** [RobinHerbots/Inputmask#2071]
**Describe the bug**
When the input mask is set on a empty field the prefix is showed incorrectly until any value is typed on it.
Correct: `R$`
What is shown: `0$`

**To Reproduce**
```
{
    alias: 'currency',
    prefix: "R$ ",
    groupSeparator: ".",
    radixPoint: ",",
    rightAlign: false
}
```

**Expected behavior**

**Live example**
https://jsfiddle.net/bqmtk1u7/10/

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 72

**Comment by zickmho** [comment]
Yes I was also having the same problem when I want to use RM for the currency prefix. It will initially shows 0M as default placeholder.. Only working fine after you start typing on it.

**Comment by RobinHerbots** [comment]
@henriquecm , @zickmho ,

This is fixed in the latest beta v5.0.0

https://jsfiddle.net/utqegy7b/
